doc/chapters/integration-guidelines/index.rst

```diff
@@ -84,3 +80,30 @@ Here is an example manifest defining Wayland access::
       ]
     }
 
+
+The role of a launcher
+======================
+
+This section describes what typical integration actions are needed to integrate SoftwareContainer with
+a launcher. For an overview of the general architecture involving a launcher and SoftwareContainer, see
+:ref:`Design <design>`.
+
+The assumed scenario in this section is that a launcher want to start an application inside the container.
+
+The launcher should do the following:
+
+ * Make the app home directory available inside the container.
+ * Set the HOME environment variable in the container point to the above directory.
+
+The above actions are performed by interacting with the SoftwareContainerAgent :ref:`D-Bus API <api>`.
```

Comment:
This ties in closely to the docs describing the launcher in the design chapter. Are there opportunities for consolidation?